new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java

```diff
@@ -0,0 +1,603 @@
+                .clusterName("cluster-1")
+                .replicas(replicaCount)
+                .serverStartState("RUNNING"))
+            .configuration(new Configuration()
+                .model(new Model()
+                    .domainType("WLS")
+                    .configMap(configMapName)
+                    .runtimeEncryptionSecret(encryptionSecretName))));
+
+    logger.info("Create domain custom resource for domainUID {0} in namespace {1}",
+        domainUid, domainNamespace);
+    assertTrue(assertDoesNotThrow(() -> createDomainCustomResource(domain),
+        String.format("Create domain custom resource failed with ApiException for %s in namespace %s",
+            domainUid, domainNamespace)),
+        String.format("Create domain custom resource failed with ApiException for %s in namespace %s",
+            domainUid, domainNamespace));
+
+
+    // wait for the domain to exist
+    logger.info("Check for domain custom resouce in namespace {0}", domainNamespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                domainUid,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(domainExists(domainUid, DOMAIN_VERSION, domainNamespace));
+
+
+    // check admin server pod exist
+    logger.info("Check for admin server pod {0} existence in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodCreated(adminServerPodName);
+
+    // check managed server pods exists
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check for managed server pod {0} existence in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodCreated(managedServerPrefix + i);
+    }
+
+    // check admin server pod is running
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkPodRunning(adminServerPodName);
+
+    // check managed server pods are running
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed server pod {0} to be ready in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkPodRunning(managedServerPrefix + i);
+    }
+
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName, domainNamespace);
+    checkServiceCreated(adminServerPodName);
+
+    // check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managed server service {0} is created in namespace {1}",
+          managedServerPrefix + i, domainNamespace);
+      checkServiceCreated(managedServerPrefix + i);
+    }
+
+    oracle.weblogic.kubernetes.utils.ExecResult result = null; 
+    try {
```

Comment:
Add this to TestConstants.java and use K8S_NODEPORT_HOST in place of host name
```
  public static final String K8S_NODEPORT_HOST = Optional.ofNullable(System.getenv("K8S_NODEPORT_HOST"))
      .orElse(InetAddress.getLocalHost().getHostName());
```

Reply:
Added following varaible to TestConstants.java 
public static final String K8S_NODEPORT_HOST = System.getenv().getOrDefault("K8S_NODEPORT_HOST",
      System.getenv("HOST"));